docs/src/pages/index.page.tsx

```diff
@@ -99,7 +99,7 @@ const AmpCard = ({ title, description, href }) => (
 const HomePage = ({ colorMode, setThemeOverride, themeOverride }) => {
   const router = useRouter();
```

Comment:
I cannot comment on line 80. Do you mind removing those dead codes?

Reply:
Oh those are actually mine and are there intentionally, 

<img width="1439" alt="CleanShot 2022-02-24 at 16 21 35@2x" src="https://user-images.githubusercontent.com/321279/155629813-f9b556b2-d119-4ef0-8098-b4fd30c62c0f.png">